src/schema/objects/extensions.yaml

```diff
@@ -0,0 +1,251 @@
+.dat:
+  name: MEG Fine-Calibration Format
+  description: |
+    A fine-calibration file.
+.ds/:
+  name: CTF MEG Dataset Folder
+  description: |
+    A folder for MEG data, typically containing a `.meg4` file for the data and a `.res4` file for the resources.
+.edf:
+  name: European Data Format
+  description: |
+    A [European data format](https://www.edfplus.info/) file.
+
+    Each recording consists of a single `.edf`` file.
+    [`edf+`](https://www.edfplus.info/specs/edfplus.html) files are permitted.
+    The capital `.EDF` extension MUST NOT be used.
+.eeg:
+  name: BrainVision Binary Data
+  description: |
+    A binary data file in the
+    [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5).
+    These files come in three-file sets, including a `.vhdr`, a `.vmrk`, and a `.eeg` file.
+.fdt:
+  name: EEGLAB FDT
+  description: |
+    A [EEGLAB](https://sccn.ucsd.edu/eeglab) file.
+
+    The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab).
+    Each recording consists of a `.set` file with an optional `.fdt` file.
+.fif:
+  name: Functional Imaging File Format
+  description: |
+    An MEG file format used by Neuromag, Elekta, and MEGIN.
+.jpg:
+  name: Joint Photographic Experts Group Format
+  description: |
+    A JPEG image file.
+.json:
+  name: JavaScript Object Notation
+  description: |
+    A JSON file.
+
+    In the BIDS specification, JSON files are primarily used as "sidecar" files, in which metadata describing "data"
+    files are encoded.
+    These sidecar files follow the inheritance principle.
+
+    There are also a few special cases of JSON files being first-order data files, such as `genetic_info.json`.
+.kdf:
+  name: KRISS KDF
+  description: |
+    A KRISS (file with extension `.kdf`) file.
+
+    Each experimental run on the KRISS system produces a file with extension `.kdf`.
+    Additional files that may be available in the same folder include
+    the digitized positions of the head points (`\_digitizer.txt`),
+    the position of the center of the MEG coils (`.chn`),
+    and the event markers (`.trg`).
+.md:
+  name: Markdown
+  description: |
+    A Markdown file.
+.mefd/:
+  name: Multiscale Electrophysiology File Format Version 3.0
+  description: |
+    A folder in the [MEF3](https://osf.io/e3sf9/) format.
+
+    Each recording consists of a `.mefd` directory.
+.mhd:
+  name: ITAB Binary Header
+  description: |
+    Produced by ITAB-ARGOS153 systems. This file a binary header file, and is generated along with a
+    raw data file with the `.raw` extension.
+.mrk:
+  name: MRK
+  description: |
+    A file containing MEG sensor coil positions.
+
+    Used by KIT, Yokogawa, and Ricoh MEG systems.
+    Successor to the `.sqd` extension for marker files.
+.nii:
+  name: Nifti
+  description: |
+    A nifti file.
+.nii.gz:
+  name: Compressed Nifti
+  description: |
+    A compressed nifti file.
```

Comment:
```suggestion
  name: NIfTI
  description: |
    A NIfTI file.
.nii.gz:
  name: Compressed NIfTI
  description: |
    A compressed NIfTI file.
```

Reply:
Is there other info worth including in the NIfTI description?

In practice, we tend to restrict NIfTIs to 3-4 dimensions, even though they can support more (8 I think?). I'm not sure if that's something we want to be explicit about or not.

Reply:
We don't really put constraints on them. If we wanted to link, we could use: https://nifti.nimh.nih.gov/

And it's up to 7 dimensions. `dim` is an array of length 8, but `dim[0]` is the number of dimensions.

Reply:
Ah, fair enough. I feel like we do try to avoid 5+D niftis in the specification, but it's not a formal restriction (i.e., someone could propose a suffix that requires 7D niftis and we could merge that). The link seems to have gone down some time between when you shared it and now, so I'm just going to skip that for now.